package.json

```diff
@@ -1,39 +1,48 @@
 {
-  "name": "bucko",
-  "author": "Tradeshift",
-  "version": "0.0.1",
-  "description": "A simple Tradeshift app that demonstrates making API calls using OAuth2",
-  "private": true,
-  "license": "FreeBSD",
-  "scripts": {
-    "start": "node ./bin/server",
-    "test": "echo \"Error: no test specified\" && exit 1"
-  },
-  "engines": {
-    "node": "0.12.0"
-  },
-  "dependencies": {
-    "body-parser": "^1.14.2",
-    "cons": "0.0.2",
-    "consolidate": "^0.13.1",
-    "debug": "^2.2.0",
-    "express": "^4.13.3",
-    "express-handlebars": "^2.0.1",
-    "express-session": "^1.12.1",
-    "fs-extra": "^0.26.3",
-    "handlebars": "^4.0.5",
-    "http": "^0.0.0",
-    "https": "^1.0.0",
-    "jade": "^1.11.0",
-    "lodash": "^3.10.1",
-    "mime": "^1.3.4",
-    "mockaroo": "^0.1.6",
-    "moment": "^2.11.0",
-    "morgan": "^1.6.1",
-    "node-rest-client": "^1.5.1",
-    "node-uuid": "^1.4.7",
-    "path": "^0.12.7",
-    "q": "^1.4.1",
-    "superagent": "^1.6.1"
-  }
+	"name": "bucko",
+	"author": "Tradeshift",
+	"version": "0.0.2",
+	"description": "A sample app for getting started developing for the Tradeshift platform",
```

Comment:
`for the Tradeshift platform` -> `on the Tradeshift platform` maybe?

Reply:
yeah